README.md

```diff
@@ -283,6 +283,53 @@ repos:
       stages: ["commit", "push"]
 ```
 
+# Pre-receive hook
```

Comment:
@L11R _I think_ this is something we don't want in our README, _yet_.  We haven't tested it yet and don't want to showcase something that hasn't been thoroughly tested.

Also this README addition is unrelated to this PR. The "Pre-commit" typos are fine though

Reply:
@zricethezav sure, understand. Will remove!